components/index.js

```diff
@@ -1,3 +1,5 @@
+// tokens
```

Comment:
Nitpicking but to stay consitant with you did below we should `Tokens`
Ok ... I'm picky 😝

Reply:
I agree, let's be consistent :)
```suggestion
// Tokens
```

Reply:
^^